assets/js/modules/subscribe-with-google/components/settings/SettingsForm.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Subscribe with Google Account Create component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { Fragment } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import StoreErrorNotices from '../../../../components/StoreErrorNotices';
+import { STORE_NAME } from '../../datastore/constants';
+import { ProductsInput, PublicationIDInput } from '../common';
+
+export default function SettingsForm() {
+	return (
+		<Fragment>
+			<StoreErrorNotices
+				moduleSlug="subscribe-with-google"
+				storeName={ STORE_NAME }
+			/>
+
+			<div className="googlesitekit-setup-module__inputs">
+				<PublicationIDInput />
+			</div>
+
+			<div className="googlesitekit-setup-module__inputs googlesitekit-setup-module__inputs--multiline">
+				<ProductsInput />
+			</div>
+		</Fragment>
+	);
+}
```

Comment:
It would be great if you could add Storybook coverage for this component so that it appears e.g. in https://google.github.io/site-kit-wp/storybook/develop/ later. Basically:
* Create a `SettingsForm.stories.js` file next to this one.
* Use `Modules/Subscribe with Google/Settings/SettingsForm` as story title.
* It could have a single story, just called "Default" or something.
* You can look at e.g. https://github.com/google/site-kit-wp/blob/develop/assets/js/modules/analytics/components/settings/SettingsForm.stories.js to see an example for such a file - the one needed here can probably be a lot simpler though.